From 64a37bf5849af931f9e82ea440f44bd876a86a2b Mon Sep 17 00:00:00 2001
From: Rob Clark <robdclark@gmail.com>
Date: Mon, 3 Jul 2017 08:34:37 -0400
Subject: [PATCH] HACK: disable emmc

Hitting some timeout which makes boot take much longer.  And
uefi/boot/rootfs partitions will be on sd-card or usb disk, etc,
so we can just ignore emmc.
---
 arch/arm/dts/dragonboard410c.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/dts/dragonboard410c.dts b/arch/arm/dts/dragonboard410c.dts
index fa348bc621..bfe7f15d5e 100644
--- a/arch/arm/dts/dragonboard410c.dts
+++ b/arch/arm/dts/dragonboard410c.dts
@@ -106,6 +106,7 @@
 			#phy-cells = <0>;
 		};
 
+/*
 		sdhci@07824000 {
 			compatible = "qcom,sdhci-msm-v4";
 			reg = <0x7824900 0x11c 0x7824000 0x800>;
@@ -115,6 +116,7 @@
 			clock = <&clkc 0>;
 			clock-frequency = <100000000>;
 		};
+*/
 
 		sdhci@07864000 {
 			compatible = "qcom,sdhci-msm-v4";
-- 
2.17.1

